extra_geom/crystfel_fmt.py

```diff
@@ -74,7 +75,7 @@ def frag_to_crystfel(fragment, p, a, ss_slice, fs_slice, dims, pixel_size):
         fs_vec=_crystfel_format_vec(fragment.fs_vec/ pixel_size),
         corner_x=c[0],
         corner_y=c[1],
-        coffset=c[2],
+        coffset=coffset,
```

Comment:
```suggestion
        coffset=fragment.corner_pos[2],
```

No need for an extra variable. :slightly_smiling_face: